site/en/blog/new-in-chrome-98/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: New in Chrome 98
+description: >
+  Chrome 98 is rolling out now! If you’ve signed up
+  for the Auto Dark Theme Origin Trial, there’s a new way to opt out at a per
+  element level. There’s now support for COLRv1, an evolution of the COLRv0
+  font format. And there’s plenty more.
+layout: 'layouts/blog-post.njk'
+date: 2022-02-01
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/Eme4o8bSchfthB5p9g2R.png'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-98
+---
+
+{% YouTube id='TODO' %}
+
+Here's what you need to know:
+
+* If you’ve tried the Auto Dark Theme Origin Trial, there’s a new way to
+  [opt out at a per element level](#autodark-opt-out).
+* There’s now support for [COLRv1](#colrv1), an evolution of the COLRv0 font
+  format.
+* And there's plenty [more](#more).
+
+I'm [Pete LePage](https://petelepage.com). Let's dive in and
+see what's new for developers in Chrome 98.
+
+## Opting out of auto-dark themes on Android {: #autodark-opt-out }
+
+In Chrome 96, we started an origin trial for
+[Auto Dark Themes on Android][dcc-autodark].
+
+With this feature, if your site is light themed and the user’s OS is in dark
+mode, the browser applies an automatically generated dark theme to your site.
+
+If you've signed up for the origin trial, and you want to opt out of the
+automatically generated dark theme for a specific page, you can use a meta tag.
+
+```html
+<meta name="color-scheme" content="only light">
+```
+
+Or you can opt-out by setting `color-scheme: only light` on the `:root`
+element.
+
+```css
+:root {
+  color-scheme: only light;
+}
+```
+
+But the main advantage of this method, is that you can opt out on a
+per-element basis, by applying a specific style to that element.
+
+```css
+.only-light,
+#my-element {
+  color-scheme: only light;
+}
+```
+
+Check out the section on [How to opt-out of Auto Dark Theme][dcc-autodark-optout]
```

Comment:
Section? I was looking for it on this page.

Reply:
modified this.